app/src/ui/app.tsx

```diff
@@ -1393,18 +1398,24 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setUpdateBannerVisibility(false)
 
   private currentPopupContent(): JSX.Element | null {
-    // Hide any dialogs while we're displaying an error
-    if (this.state.errors.length) {
-      return null
-    }
-
     const popup = this.state.currentPopup
 
     if (!popup) {
       return null
     }
 
-    const onPopupDismissedFn = this.getOnPopupDismissedFn(popup.type)
+    if (popup.id === undefined) {
```

Comment:
Unless I'm missing something, this could happen with the feature flag disabled and therefore we'll send many non-fatal errors to Sentry. Could we check the feature flag here too?
```suggestion
    if (popup.id === undefined && enableStackedPopups()) {
```

(If you accept this suggestion you will need to import the feature flag at the top)

Reply:
I am going to resolve this by just making all popups now have an id (whether flag is enabled or not)